xml/article_sap_trento.xml

```diff
@@ -0,0 +1,535 @@
+                </para>
+                <screen>&trentoserver.prompt;curl -sfL https://raw.githubusercontent.com/trento-project/trento/main/install-server.sh | bash</screen>
+            </step>
+            <step>
+                <para>
+                    Add the absolute path to your private SSH key from <xref linkend="st-trento-server-ssh"/> (usually
+                    <filename>/home/&trentoadmin;/.ssh/id_rsa</filename>):
+                </para>
+                <screen>Please provide the path of the runner private key: /home/&trentoadmin;/.ssh/id_rsa</screen>
+                <para>
+                    The script will install Helm and other components.
+                    After the installation was successful, the script will respond with:
+                </para>
+                <screen>Trento Server installed!</screen>
+            </step>
+            <step>
+                <para>
+                    Check whether the Trento server is up and running:
+                </para>
+                <screen>&trentoserver.prompt;sudo /usr/local/bin/kubectl get pods
+NAME                                      READY   STATUS    RESTARTS   AGE
+svclb-trento-server-web-collector-x2hmd   1/1     Running   0          42m
+trento-server-consul-server-0             1/1     Running   0          42m
+trento-server-consul-qv65d                1/1     Running   0          42m
+trento-server-postgresql-0                1/1     Running   0          42m
+trento-server-ara-77fb85b46d-mb756        1/1     Running   0          42m
+trento-server-web-df665479-rlghw          1/1     Running   2          42m
+trento-server-runner-5cdc4995b6-f96vb     1/1     Running   1          42m</screen>
+            </step>
+            <step>
+                <para>
+                    To ensure your Trento agents are able to connect to the Trento server,
+                    check the network ports:
+                </para>
+                <screen>&trentoserver.prompt;sudo /usr/local/bin/kubectl describe pod trento-server-web-df665479-rlghw | grep -i port    Ports:         8080/TCP, 8081/TCP
+    Host Ports:    0/TCP, 0/TCP
+      --db-port
+      --port
+      --collector-port</screen>
+            </step>
+            <step>
+                <para>
+                    Log out from your Trento server.
+                </para>
+            </step>
+            <step>
+                <para>
+                    On your workstation, download the Trento server generated public SSH key
+                    and save it in your <filename>~/.ssh</filename> directory:
+                </para>
+                <screen>&prompt.user;scp -i ~/.ssh/id_rsa_cloud &trentoadmin;@&trentoservername;:/home/&trentoadmin;/.ssh/id_rsa.pub ~/.ssh/id_rsa_trento_server.pub</screen>
+                <para>
+                    You will need the Trento generated public SSH key for your Trento agents later.
+                </para>
+            </step>
```

Comment:
> But how the customer transfer the key from one server to another in their environment should be up to them.

Maybe. :thinking: I've added this step mainly because of our target group. As far as I understood it, we not always deal with Linux experts, but also people coming from a Windows background. For them, it might not be clear.